javamds/MDSplus_TreeNode.h

```diff
@@ -307,6 +325,14 @@ JNIEXPORT void JNICALL Java_MDSplus_TreeNode_turnOn
 JNIEXPORT jboolean JNICALL Java_MDSplus_TreeNode_isOn
   (JNIEnv *, jclass, jint, jlong);
 
+/*
```

Comment:
Are you sure you want to implement isParentOff? since you already have a isOn, the parent version should be changed to isParentOn returning the inverse.

You need to change this where you used it... if you are using it already.